### ISSUE TITLE
New load condition: Specific class is in group

### ISSUE THREAD

**Issue by Mystream90** [WeakAuras/WeakAuras2#4158]
**Is your feature request related to a problem? Please describe.**

Not a problem, maybe it is too specific. Lets say I have spell I wanna track which can only used by a specific class in a group/raid. Currently it is not possible to unload auras when a specific class is not in a group. Basically it is the same as PlayerUnit Info > Unit Characteristics > Smart Group > Class: Warlock (e.g.) but in the load tab.

**Describe the solution you'd like**
Basically it is the same as PlayerUnit Info > Unit Characteristics > Smart Group > Class: Warlock (e.g.) but in the load tab.

**Describe alternatives you've considered**
Using "PlayerUnit Info > Unit Characteristics > Smart Group > Class: Warlock (e.g.)" as a trigger but when having a lot of auras with the same trigger it takes too much (space?)

**Additional context**

**Comment by InfusOnWoW** [comment]
That far too specifric.
